.gitignore

```diff
@@ -0,0 +1,3 @@
+/bin/
```

Comment:
Could be something like [zap-api-java .gitignore](https://github.com/zaproxy/zap-api-java/blob/develop/.gitignore)? (except for subprojects entries which are not required in this case)